### PR TITLE
Dev/ngis spec v3.22/new gel1004 message

### PR DESCRIPTION
**Pull request by ReubenMonaghan**

### CODE REVIEW

File: platerplotter/platerplotter/holding_rack_manager.py [hunk 1]
```diff
@@ -167,8 +167,8 @@ def assign_well(self, request, sample, well):
 							sample.laboratory_sample_id + " to a new rack.")
 		elif self.holding_rack.holding_rack_type == "Family":
 			matching_proband_sample_found = True
-			# all new family samples must have a matching proband
-			if sample.is_repeat == "New":
+			# all new family samples must have a matching proband except sample_delivery_mode="Family Only" samples
+			if sample.is_repeat == "New" and sample.sample_delivery_mode != "Family Only":
```
Comment (joelarkman) [inline platerplotter/platerplotter/holding_rack_manager.py:171]: Just trying to think through all the various combinations to see if there are any scenarios where these ANDs should actually be ORs but I think we are okay - I guess we're only needing to add an extra exclusion to when this code used to be triggered.

**Review by joelarkman — CHANGES_REQUESTED** [review]
Hi Reuben,

These changes are looking good and pretty much ready to merge, i've just added a few thoughts, questions and one suggestion that I wanted to run past you before we action the pull request.

Many thanks,
Joe

File: platerplotter/platerplotter/models.py [hunk 2]
```diff
@@ -145,6 +145,8 @@ class Sample(models.Model):
 	tissue_type = models.CharField(max_length=50, choices=(("Normal or Germline sample", "Normal or Germline sample"),
 		("Liquid tumour sample", "Liquid tumour sample"), ("Solid tumour sample", "Solid tumour sample"),
 		("Abnormal tissue sample", "Abnormal tissue sample"), ("Omics sample", "Omics sample")))
+	sample_delivery_mode = models.CharField(max_length=50, blank=True, default="Standard", choices=(("Tumour First", "Tumour First"),
```
[the remark this answers is not in the document]
Comment (ReubenMonaghan) [reply platerplotter/platerplotter/models.py:148]: I think I just need to check importing old Gel1004 messages work on the BI test server, but I ran the migrate command with no issues.

**Review by ReubenMonaghan — COMMENTED** [review]
Need to remove "" from accepted_values in check_sample_delivery_mode() views.py line 190.

**Review by joelarkman — APPROVED** [review]
Hi Reuben,

Thanks for making the changes, this looks good and ready to merge to me.

Many thanks,
Joe

**Pull request merged**